com.unity.render-pipelines.universal/Runtime/Passes/DepthOnlyPass.cs

```diff
@@ -50,15 +55,28 @@ public DepthOnlyPass(RenderPassEvent evt, RenderQueueRange renderQueueRange, Lay
 
             this.allocateDepth = true;
             this.shaderTagId = k_ShaderTagId;
+            this.useDepthPriming = useDepthPriming;
         }
 
         public override void OnCameraSetup(CommandBuffer cmd, ref RenderingData renderingData)
         {
             if (this.allocateDepth)
                 cmd.GetTemporaryRT(depthAttachmentHandle.id, descriptor, FilterMode.Point);
             var desc = renderingData.cameraData.cameraTargetDescriptor;
-            ConfigureTarget(new RenderTargetIdentifier(depthAttachmentHandle.Identifier(), 0, CubemapFace.Unknown, -1), GraphicsFormat.DepthAuto, desc.width, desc.height, 1, true);
-            ConfigureClear(ClearFlag.All, Color.black);
+
+            // When depth priming is in use the camera target should not be overridden so the Camera's MSAA depth attachment is used.
+            if (this.useDepthPriming)
+            {
+                ConfigureTarget(new RenderTargetIdentifier(depthAttachmentHandleMSAA.Identifier(), 0, CubemapFace.Unknown, -1), GraphicsFormat.DepthAuto, desc.width, desc.height, 1, true);
+            }
+            // When not using depth priming the camera target should be set to our non MSAA depth target.
+            else
+            {
+                ConfigureTarget(new RenderTargetIdentifier(depthAttachmentHandle.Identifier(), 0, CubemapFace.Unknown, -1), GraphicsFormat.DepthAuto, desc.width, desc.height, 1, true);
+            }
+
+            // Only clear depth here so we don't clear any bound color target. It might be unused by this pass but that doesn't mean we can just clear it. (e.g. in case of overlay cameras + depth priming)
+            ConfigureClear(ClearFlag.Depth, Color.black);
```

Comment:
@manuele-bonanno since you are working on MSSA and depth "prepass" we have today.